### ISSUE TITLE
Filter vulernabilities without fixes

### ISSUE THREAD

**Issue by lee5i3** [anchore/grype#175]
We would like to use this to see any issues, however if no fix is available we definitely do not want to fail our builds

Can we get a way to only fail if there is a fix, something like this

grype <image> --fail-on medium **--only-fixed**

We don't want it to exit code 1 if there no fix

**Comment by outrunthewolf** [comment]
I take it this isnt a thing yet then?

**Comment by luhring** [comment]
@outrunthewolf Correct, it's not a thing yet. But it makes sense, and I'd love to see this get implemented. I can bring it up for a prioritization discussion with the core team; but, we also welcome PRs 👍

**Comment by luhring** [comment]
I'm wondering if there's an elegant way to extend the new [**ignore rules** functionality](https://github.com/anchore/grype#specifying-matches-to-ignore) (from the implementation of #198) to solve this problem... 🤔
